Docs/user_guide/get_help.md

```diff
@@ -0,0 +1,31 @@
+# Get Help
+
+[Contact the Slicer community or commercial partners](about.html#contact-us) if you have any questions, bug reports, or enhancement requests.
+
+## I need help in using Slicer
+
+- I have a question about how to use Slicer for a particular task or I'm not sure things are working right:
+  - Type your queston into Google. There is a good chance that your question has been asked and anwered before and all questions ever asked about Slicer are publicly available and indexed by Google.
```

Comment:
Unclear if some of the first DOC commits are intended for this PR as they were originally included in https://github.com/Slicer/Slicer/pull/5268

Still the issue of “queston” here as mentioned in a review on that other PR.

Reply:
Good catch! It seems that I accidentally included that same commit in this PR, too (was in the same integration branch). I'll fix your comments before merging. Thank you for the review!